pgsync/sync.py

```diff
@@ -418,10 +430,20 @@ def should_skip_event(self, event: dict) -> bool:
                 that are skippable as per settings.SKIP_NULL_KEYS
         """
 
-        logger.debug(f"validating event: {event}")
-
-        if (event['schema'], event['table']) not in self.valid_tables:
-            logger.debug(f"skipping event for {event['schema']}.{event['table']} as it's not in the configured pgsync JSON schema")
+        logger.debug(f"should_skip_event: validating event: {event}")
+
+        if (event['schema'], event['table']) in self._gist_schema_attributes:
+            if CHANGED_FIELD_NAME in event and event[CHANGED_FIELD_NAME] is not None:
+                skip_status = True
+                for column in event[CHANGED_FIELD_NAME]:
+                    if column in self._gist_schema_attributes[(event['schema'], event['table'])]:
+                        skip_status = False
+                        break
+                if skip_status:
+                    logger.debug(f"should_skip_event: skip event as [ {event[CHANGED_FIELD_NAME]} ] was modifyed that not configured pgsync JSON schema for {event['schema']}.{event['table']} {self._gist_schema_attributes[(event['schema'], event['table'])]}")
```

Comment:
```python
"skipping the event with these fields [{event[CHANGED_FIELD_NAME]}] modified since they are not in the configured pgsync JSON schema"
```

Reply:
it's not informative when spiking message

Reply:
full log message should be:
```python
logger.debug(f"should_skip_event: skipping the event with these fields [{event[CHANGED_FIELD_NAME]}] modified since they are not in the configured pgsync JSON schema for {event['schema']}.{event['table']} {self._gist_schema_attributes[(event['schema'], event['table'])]}")
```